src/utils/register-foreign-multiple.coffee

```diff
@@ -0,0 +1,65 @@
+program = require 'commander'
+msgflo_nodejs = require 'msgflo-nodejs'
+fs = require 'fs'
+path = require 'path'
+foreigner = require '../foreign-participant'
```

Comment:
Yes, but the `foreign-participant` lib provides the `#ROLE` mapping magic we need here